src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,26 +1,107 @@
 package frc.robot;
```

Comment:
This file will be long, but I'm not sure I have a good alternative. I'd at least put the vision init and swerve init code in different functions.

Reply:
e9883f326f94f9855e29c096bf204860a2349355